libs/platform/src/lib/components/form/multi-combobox/multi-combobox/multi-combobox.component.ts

```diff
@@ -0,0 +1,346 @@
+import {
+    AfterViewInit,
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    ElementRef,
+    Host,
+    Inject,
+    Injector,
+    Input,
+    OnInit,
+    Optional,
+    Self,
+    SkipSelf,
+    TemplateRef,
+    ViewChild,
+    ViewEncapsulation
+} from '@angular/core';
+import { NgControl, NgForm } from '@angular/forms';
+import { CdkConnectedOverlay } from '@angular/cdk/overlay';
+import { A, DOWN_ARROW, ENTER, LEFT_ARROW, RIGHT_ARROW, SPACE, UP_ARROW } from '@angular/cdk/keycodes';
+import { Direction } from '@angular/cdk/bidi';
+import { takeUntil } from 'rxjs/operators';
+
+import {
+    closestElement,
+    DialogConfig,
+    DynamicComponentService,
+    KeyUtil,
+    RtlService,
+    TokenizerComponent
+} from '@fundamental-ngx/core';
+import { DATA_PROVIDERS, DataProvider, ListDataSource } from '../../../../domain/data-source';
+import { FormFieldControl } from '../../form-control';
+import { BaseMultiCombobox, MultiComboboxSelectionChangeEvent } from '../commons/base-multi-combobox';
+import { OptionItem, SelectableOptionItem } from '../../../../domain';
+import { MultiComboboxMobileComponent } from '../multi-combobox-mobile/multi-combobox/multi-combobox-mobile.component';
+import { MULTICOMBOBOX_COMPONENT } from '../multi-combobox.interface';
+import { FormField } from '../../form-field';
+import { ComboboxConfig } from '../../combobox/combobox.config';
+
+@Component({
+    selector: 'fdp-multi-combobox',
+    templateUrl: './multi-combobox.component.html',
+    styleUrls: ['./multi-combobox.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    encapsulation: ViewEncapsulation.None,
+    providers: [{ provide: FormFieldControl, useExisting: MultiComboboxComponent, multi: true }]
+})
+export class MultiComboboxComponent extends BaseMultiCombobox implements OnInit, AfterViewInit {
+    @Input()
+    get disabled(): boolean {
```

Comment:
`@Input disable` already exist in `BaseInput` component
Maybe we need to use it?